end-to-end-tests/src/test/java/tech/pegasys/peeps/consensus/qbft/quorumbesu/QbftWithGasAndBlockRewardTest.java

```diff
@@ -0,0 +1,133 @@
+import tech.pegasys.peeps.node.Web3Provider;
+import tech.pegasys.peeps.node.Web3ProviderType;
+import tech.pegasys.peeps.node.verification.ValueReceived;
+import tech.pegasys.peeps.node.verification.ValueSent;
+import tech.pegasys.peeps.signer.SignerConfiguration;
+
+import java.util.List;
+
+import org.apache.tuweni.crypto.SECP256K1.KeyPair;
+import org.apache.tuweni.eth.Address;
+import org.apache.tuweni.units.ethereum.Wei;
+import org.junit.jupiter.api.Test;
+
+public class QbftWithGasAndBlockRewardTest extends NetworkTest {
+  private final Wei gasPrice = Wei.valueOf(1);
+
+  private final Wei blockReward = Wei.valueOf(1);
+
+  private Web3Provider quorumNode1;
+  private Web3Provider quorumNode2;
+  private Web3Provider besuNode1;
+  private Web3Provider besuNode2;
+
+  private final SignerConfiguration signer = FixedSignerConfigs.ALPHA;
+
+  private final Address miningBeneficiary =
+      Address.fromHexString("0xab5801a7d398351b8be11c439e05c5b3259aec9b");
+
+  @Override
+  protected void setUpNetwork(final Network network) {
+    besuNode1 = network.addNode("besu1", KeyPair.random(), Web3ProviderType.BESU, gasPrice);
+    besuNode2 = network.addNode("besu2", KeyPair.random(), Web3ProviderType.BESU, gasPrice);
+    quorumNode1 = network.addNode("quorum1", KeyPair.random(), Web3ProviderType.GOQUORUM, gasPrice);
+    quorumNode2 = network.addNode("quorum2", KeyPair.random(), Web3ProviderType.GOQUORUM, gasPrice);
+
+    network.addBlockRewardTransition(20, blockReward);
+    network.addMiningBeneficiaryTransition(40, miningBeneficiary);
+
+    network.set(ConsensusMechanism.QBFT, besuNode1, besuNode2, quorumNode1, quorumNode2);
+
+    network.addSigner(signer.name(), signer.resources(), besuNode1, gasPrice);
+  }
+
+  @Test
+  public void validatorReceivesGasRewards() {
+    verify().consensusOnBlockNumberIsAtLeast(1);
+
+    final Address sender = signer.address();
+    final Address receiver = Account.BETA.address();
+    final Wei transferAmount = Wei.valueOf(5000L);
+
+    verify().consensusOnValueAt(sender, receiver);
+
+    final Wei senderStartBalance = execute(besuNode1).getBalance(sender);
+    final Wei receiverStartBalance = execute(besuNode2).getBalance(receiver);
+
+    var receipt = execute(signer).transferTo(receiver, transferAmount);
+
+    await().consensusOnTransactionReceipt(receipt);
+
+    verifyOn(besuNode2)
+        .transition(
+            new ValueSent(sender, senderStartBalance, receipt),
+            new ValueReceived(receiver, receiverStartBalance, transferAmount));
+
+    var validatorBalances =
+        List.of(
+            execute(besuNode1).getBalance(besuNode1.address()),
+            execute(besuNode1).getBalance(besuNode2.address()),
+            execute(besuNode1).getBalance(quorumNode1.address()),
+            execute(besuNode1).getBalance(quorumNode2.address()));
+
+    assertThat(validatorBalances)
+        .containsOnly(Wei.valueOf(21000), Wei.valueOf(0), Wei.valueOf(0), Wei.valueOf(0));
```

Comment:
Could extract 21000 to a var to make this clear is it is validator1's eth transfer gas cost reward?